pramen/project/Versions.scala

```diff
@@ -93,13 +93,15 @@ object Versions {
     }
   }
 
-  def getAbrisDependency(scalaVersion: String): ModuleID = {
-    // According to this: https://docs.delta.io/latest/releases.html
-    val abrisVersion = scalaVersion match {
-      case version if version.startsWith("2.11.") => "5.1.1"
-      case version if version.startsWith("2.12.") => "5.1.1"
-      case version if version.startsWith("2.13.") => "6.0.0"
-      case _                                      => throw new IllegalArgumentException(s"Scala $scalaVersion not supported for Abris dependency.")
+  def getAbrisDependency(sparkVersion: String): ModuleID = {
+    // According to this: https://github.com/AbsaOSS/ABRiS?tab=readme-ov-file#supported-versions
+    val abrisVersion = sparkVersion match {
+      case version if version.startsWith("2.4.") => "5.1.1"
+      case version if version.startsWith("3.0.") => "5.1.1"
+      case version if version.startsWith("3.1.") => "5.1.1"
+      case version if version.startsWith("3.2.") => "6.1.1"
```

Comment:
- `6.1.1` is really only for Spark `3.2.0` specifically
- For Spark `3.2.x (x > 0)` the latest Abris version should be used

All this mess is caused by Spark breaking a compatibility: https://github.com/AbsaOSS/ABRiS/issues/280

Reply:
Awesome, thanks!

Reply:
Fixed